cypress/integration/session-recording.spec.js

```diff
@@ -8,7 +8,7 @@ describe('Session recording', () => {
             method: 'POST',
             url: '**/decide/*',
             response: {
-                config: { enable_collect_everything: true },
+                config: { enable_collect_everything: false },
```

Comment:
Q: Why this change?

Reply:
Intended to make sure that session recording works even if autocapture is off